sunpy/net/attr.py

```diff
@@ -7,10 +7,6 @@
 (a AND b AND c) OR (d AND e).
 
 Walkers are used to traverse the tree that results from combining attributes.
-They are implemented using sunpy.util.multimethod. Multimethods are functions
-that are not assigned to classes but still dispatch by type of one or more
-of their arguments. For more information about multimethods, refer to
-sunpy.util.multimethod.
```

Comment:
We should probably re-write this to explain they use singledispatch instead.

Reply:
Can someone else who know more about this provide some words (either as a comment here or push to my branch)?